components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -458,6 +489,15 @@ std::string AIChatTabHelper::BuildLlama2Prompt(std::string user_message) {
   // Always use a generic system message
   std::string system_message =
       l10n_util::GetStringUTF8(IDS_AI_CHAT_LLAMA2_SYSTEM_MESSAGE_GENERIC);
+  base::Time::Exploded exploded;
+  base::Time::Now().LocalExplode(&exploded);
+  std::string date_string = base::StringPrintf(
+      "%02d:%02d %s, %s %02d, %d", exploded.hour, exploded.minute,
+      GetDayOfWeekString(exploded.day_of_week).c_str(),
+      GetMonthString(exploded.month - 1).c_str(), exploded.day_of_month,
```

Comment:
You may not need to create helpers here. There are some helpful functions in this file, but more specifically, this function could be used: https://source.chromium.org/chromium/chromium/src/+/main:base/i18n/time_formatting.h;l=100?q=base%2Fi18n%2Ftime_formatting.h

Reply:
yeah that seems much better 👍 @nvonpentz brought it up as well, I'm going to switch to that and remove the helpers